### PR TITLE
CPS: Add instructions for X11

### PR DESCRIPTION
**Pull request by seisman**
https://github.com/seismo-learn/seismology101/issues/90

### CODE REVIEW

File: source/cps/install.rst [hunk 1]
```diff
@@ -20,8 +20,24 @@
         $ mkdir -p ~/src/
         $ mv PROGRAMS.330 ~/src/CPS
 
+3.  安装依赖
 
-3.  配置:
+    编译 CPS 源码需要安装 X11 库文件::
```
Comment (core-man) [inline source/cps/install.rst:25]: Windows?

Comment (seisman) [reply source/cps/install.rst:25]: Do you mean WSL users?

Comment (core-man) [reply source/cps/install.rst:25]: yes.

Comment (seisman) [reply source/cps/install.rst:25]: It's just the same as normal Linux distros.

Comment (core-man) [reply source/cps/install.rst:25]: really? I remember one student cannot run `taup` when they use wsl without x server.

Comment (seisman) [reply source/cps/install.rst:25]: See https://github.com/seismo-learn/seismology101/issues/90#issuecomment-765020467

**Review by core-man — APPROVED** [review]

**Pull request merged**